docs/products/opensearch/concepts/access_control.rst

```diff
@@ -0,0 +1,84 @@
+Access control
+==============
+
+Aiven for OpenSearch supports index-level access control lists (ACL) to control permissions. This approach allows you to limit the operations that are available to specific connections and to restrict access to certain data sets, which improves the security of your data.
+
+You can grant the following permissions:
+
+* ``deny``: no access
+* ``admin``: full access to APIs and documents 
+* ``readwrite``: full access to documents
+* ``read``: allow only searching and retrieving documents
+* ``write``: allow updating, adding, and deleting documents
+
+Rules are defined separately for each user as ``pattern/permission`` combinations. The ``pattern`` defines the indexes that the permission applies to. Patterns are glob-style, where ``*`` matches any number of characters and ``?`` matches any character. 
+
+When multiple rules match, they are applied in the order listed above. If no rules match, access is denied.
+
+Example
+-------
+
+As an example, we can use the following set of rules:
+
+* ``logs_*/read``
+* ``events_*/write``
+* ``logs_2018*/deny``
+* ``logs_201901*/read``
+* ``logs_2019*/admin``
+
+This set would allow the service user to
+
+* add documents to ``events_2018`` (second rule),
+* retrieve and search documents from ``logs_20171230`` (first rule),
+* gain full access to ``logs_20190201`` (fifth rule), and
+* gain full access to ``logs_20190115`` (fifth rule, as the ``admin`` permission gets higher priority than the ``read`` permission in the fourth rule.
+
+The same set would deny the service user to
+
+* gain any access to ``messages_2019`` (no matching rules),
+* read or search documents from ``events_2018`` (the second rule only grants ``write`` permission), and
+* write to or use the API for ``logs_20171230`` (the first rule only grants ``read`` permission).
+
+.. note::
```

Comment:
I wonder if this note does not sit together with the ordered list above that introduces the types of permissions (start line 6).

Reply:
True, it probably is better there - that's now moved ;)